src/commands/block/get.js

```diff
@@ -41,7 +41,7 @@ GetCommand.args = [
 		name: 'blockIds',
 		required: true,
 		description: 'Comma-separated block ID(s) to get information about.',
-		parse: input => input.split(','),
+		parse: input => input.split(',').filter(val => val),
```

Comment:
I like `.filter(Boolean)` for this kind of check.

Reply:
something like
`.filter(isString)`?